workers/unity/Assets/Gdk/Core/Components/WorldCommandsTranslation.cs

```diff
@@ -285,53 +308,56 @@ public override void RegisterWithDispatcher(Dispatcher dispatcher)
 
         public void OnCreateEntityResponse(CreateEntityResponseOp op)
         {
+            CreateEntityRequest request = requestIdToCreateEntityRequest[op.RequestId.Id];
```

Comment:
A slightly better pattern may be:

```
CreateEntityRequest request;
if (!requestIdToCreateEntityRequest.TryGetValue(op.RequestId.Id, out request) 
{
	// Error handling
}
requestIdToCreateEntityRequest.Remove(op.RequestId.Id
```

Similarly for the other types of commands below.

Reply:
@samiwh suggested removing that because it should never happen, and if we wanted to check for errors that should never happen consistently then a lot would have to be changed in the GDK.

Reply:
I do agree that it should never happen, but I still feel that we should do error handling here (maybe I'm just being overly defensive)

Reply:
If there's no more comments on this I'll leave it as-is I think. Sami's point sounds reasonable to me (verifying things here and not in other similar places is weird)